### PR TITLE
Made Escape Room solution more readable

### PR DESCRIPTION
**Pull request by SansPapyrus683**
Removed that atrocious template, and also added some whitespace to make the code easier on the eyes.

### CODE REVIEW

File: solutions/CCC 20-escape.mdx [hunk 1]
```diff
@@ -14,232 +14,40 @@ So we essentially have a directed graph with vertices $1\ldots 10^6$ where there
 Thus, we can simply start DFSing from vertex $1$ and mark all the vertices that we visit in the boolean array `done`. If we set `done[N*M]=true`, then a path exists, and we can terminate after printing "yes." Note that in the code below, `todo[x]` denotes the adjacency list of `x`.
 
 ```cpp
-#include <bits/stdc++.h>
```
Comment (HarryWangATX) [inline solutions/CCC 20-escape.mdx:17]: stdc++ is fine.

File: solutions/CCC 20-escape.mdx [hunk 2]
```diff
@@ -14,232 +14,40 @@ So we essentially have a directed graph with vertices $1\ldots 10^6$ where there
...
+int M, N;
+vector<int> todo[1000005];
 bool done[1000005];
 
 void dfs(int x) {
-	if (done[x]) return;
-	if (x == N*M) {
-		ps("yes");
+	if (done[x]) {
+        return;
```
Comment (HarryWangATX) [inline solutions/CCC 20-escape.mdx:28]: fix formatting

File: solutions/CCC 20-escape.mdx [hunk 3]
```diff
@@ -14,232 +14,40 @@ So we essentially have a directed graph with vertices $1\ldots 10^6$ where there
...
+    }
+	if (x == N * M) {
+		cout << "yes" << endl;
 		exit(0);
 	}
 	done[x] = 1;
-	trav(t,todo[x]) dfs(t);
+    for (int& t : todo[x]) {
```
Comment (HarryWangATX) [inline solutions/CCC 20-escape.mdx:35]: use tabs

File: solutions/CCC 20-escape.mdx [hunk 4]
```diff
@@ -14,232 +14,40 @@ So we essentially have a directed graph with vertices $1\ldots 10^6$ where there
...
+    for (int i = 1; i < M + 1; i++) {
+        for (int j = 1; j < N + 1; j++) {
+            int x;
+            cin >> x;
+            todo[i * j].push_back(x);
+        }
+    }
 	dfs(1);
-	ps("no");
+	cout << "no" << endl;
```
Comment (HarryWangATX) [inline solutions/CCC 20-escape.mdx:51]: tabs here as well

File: solutions/CCC 20-escape.mdx [hunk 5]
```diff
@@ -14,232 +14,40 @@ So we essentially have a directed graph with vertices $1\ldots 10^6$ where there
...
-	// throws exception when do smth illegal
-	// ex. try to read letter into int
-	if (sz(s)) setIn(s+".in"), setOut(s+".out"); // for USACO
-}
+using namespace std;
 
-int M,N;
-vi todo[1000005];
+int M, N;
+vector<int> todo[1000005];
```
Comment (HarryWangATX) [inline solutions/CCC 20-escape.mdx:23]: vi is fine.

**Review by HarryWangATX — CHANGES_REQUESTED** [review]

File: solutions/CCC 20-escape.mdx (hunk 1 above)
Comment (SansPapyrus683) [reply solutions/CCC 20-escape.mdx:17]: I mean, it only adds one line to the code.

**Comment by HarryWangATX** [comment]
please use the template we argreed upon in #803 

```cpp
#include <bits/stdc++.h>
using namespace std;

using ll = long long;
using vi = vector<int>;
using pi = pair<int,int>;

#define sz(x) (int)(x).size()
#define pb push_back
#define all(x) begin(x), end(x)
#define f first
#define s second

int main() {
    cin.tie(0)->sync_with_stdio(0); 
    // freopen("name.in", "r", stdin); 
    // freopen("name.out", "w", stdout);

}
```

File: solutions/CCC 20-escape.mdx (hunk 2 above)
Comment (SansPapyrus683) [reply solutions/CCC 20-escape.mdx:28]: How so?

Comment (HarryWangATX) [reply solutions/CCC 20-escape.mdx:28]: ur using spaces

File: solutions/CCC 20-escape.mdx (hunk 1 above)
Comment (HarryWangATX) [reply solutions/CCC 20-escape.mdx:17]: use the template agreed upon in #803

File: solutions/CCC 20-escape.mdx (hunk 2 above)
Comment (HarryWangATX) [reply solutions/CCC 20-escape.mdx:28]: also braces are messed up

**Comment by SansPapyrus683** [comment]
Wait, so is using the template 100% required? Because I like to use as few macros as possible personally, and in this case not using the templates doesn't make the code bloated at all.

**Comment by HarryWangATX** [comment]
> Wait, so is using the template 100% required? Because I like to use as few macros as possible personally, and in this case not using the templates doesn't make the code bloated at all.

yes for consistency sake

**Comment by HarryWangATX** [comment]
well actually no, ignore me

**Comment by HarryWangATX** [comment]
but use the ones present there if applicable

File: solutions/CCC 20-escape.mdx (hunk 2 above)
Comment (SansPapyrus683) [reply solutions/CCC 20-escape.mdx]: Yeah, I fixed the spaces problem.
Also, how are the braces messed up?

**Review by thecodingwizard — APPROVED** [review]

File: solutions/CCC 20-escape.mdx [hunk 6]
```diff
@@ -15,231 +15,38 @@ Thus, we can simply start DFSing from vertex $1$ and mark all the vertices that
...
 }
 
 int main() {
-	setIO(); re(M,N);
-	FOR(i,1,M+1) FOR(j,1,N+1) {
-		int x; re(x);
-		todo[i*j].pb(x);
+	cin.tie(0)->sync_with_stdio(0);
+	cin >> M >> N;
+	for (int i = 1; i < M + 1; i++) {
```
Comment (bqi343) [inline solutions/CCC 20-escape.mdx:42]: use `<= M`, `<= N`

**Review by bqi343 — CHANGES_REQUESTED** [review]

**Review by HarryWangATX — APPROVED** [review]

**Pull request merged**